cli_main.py

```diff
@@ -447,7 +447,13 @@ def main(
     params = parse_docs(analysis_callable)[2]  # Index [2] for paramaters
     for param_name, dictionary in params.items():
         if "AtomGroup" in dictionary['type']:
-            analysis_kwargs[param_name] = u.select_atoms(analysis_kwargs[param_name])
+            sel = u.select_atoms(analysis_kwargs[param_name])
+            if len(sel) > 0:
+                analysis_kwargs[param_name] = sel
+            else:
+                raise ValueError("AtomGroup `-{}` with selection `{}` does not "
```

Comment:
I think `ValueError` is the best to raise unless we wish to add a defined error of our own, such as: `UserInputError`.

What are your thoughts?

Reply:
I think it is a bit to early for introducing our own errors. But lets keep this in mind if for later.

Reply:
Perfect, so I am merging :wink: